File: pinyougou-pay-interface/src/main/java/com/pinyougou/pay/service/PayService.java
package com.pinyougou.pay.service;

import java.util.Map;

public interface PayService {
    //1.向微信后台发出下单请求
    Map createNative(String out_trade_no, String money);

    //2.向后台发出查询订单的请求
    Map queryPayStatus(String out_trade_no);
}

File: pinyougou-seckill-web/src/main/webapp/js/service/payService.js
app.service("payService",function ($http) {
    //1.向微信后台发出下单请求
    this.createNative=()=>{
        return $http.get("./pay/createNative.do");
    };
    //2.向微信后台发出查询订单请求
    this.queryPayStatus=(out_trade_no)=>{
        return $http.get("./pay/queryPayStatus.do?out_trade_no=" + out_trade_no);
    }
});